lessons/static/css/commonlesson.css

```diff
@@ -793,11 +793,6 @@ th a:link, th a:visited, th a:hover, th a:active {
 }
 */
 
-.level-icon:before {
-    font: normal normal normal 14px/1 FontAwesome;
-    font-size: inherit;
-}
```

Comment:
its possible these styles were applied by css class `fa` in an older version of font awesome, so when we moved to a newer version, these stopped being the right styles to apply. this is just a guess though 🤷‍♂️ in either case, the approach we're moving toward seems more future-proof than the one you're removing. 👍